### PR TITLE
issue #909: Moodle 4.5 support

### PR DESCRIPTION
**Pull request by dmitriim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,8 @@ Dataflows is a generic workflow and processing engine which can be configured to
 
 | Moodle version | Branch           | PHP       |
 |----------------|------------------|-----------|
-| Moodle 4.1+    | MOODLE_41_STABLE | 7.4 | 8.0 |
+| Moodle 4.1-4.2 | MOODLE_41_STABLE | 7.4       |
+| Moodle 4.5     | MOODLE_45_STABLE | 8.1 - 8.3 |
```
Comment (tuanngocnguyen) [inline README.md:34]: Should it be MOODLE_4**0**1_STABLE and  MOODLE_4**0**5_STABLE

Comment (dmitriim) [reply README.md:34]: yeah true.  Mistake in MOODLE_401_STABLE

Comment (dmitriim) [reply README.md]: Fxied

File: version.php [hunk 2]
```diff
@@ -28,7 +28,7 @@
 $plugin->version = 2024101000;
 $plugin->release = 2024101000;
 $plugin->requires = 2022112800;    // Our lowest supported Moodle (3.3.0).
```
Comment (tuanngocnguyen) [inline version.php:30]: perhaps, updating the minimum required version as well as we cut it off at Moodle 4.5

Comment (tuanngocnguyen) [reply version.php:30]: add version should be increased

Comment (dmitriim) [reply version.php]: done

**Comment by tuanngocnguyen** [comment]
Look good to me

**Pull request merged**